components/atom/tag/src/Actionable/index.js

```diff
@@ -49,13 +54,28 @@ ActionableTag.propTypes = {
   iconPlacement: PropTypes.oneOf([LEFT_ICON_PLACEMENT, RIGHT_ICON_PLACEMENT]),
   onClick: PropTypes.func,
   target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top']),
+  rel: PropTypes.arrayOf(
+    PropTypes.oneOf([
```

Comment:
replace by: PropTypes.oneOf(Object.values(LINK_TYPES)) so there is no need to modify this line if you add more LINK_TYPES in the future.